monkey/monkey_island/cc/resources/agent_logs.py

```diff
@@ -23,7 +23,7 @@ def get(self, agent_id: AgentID):
             log_contents = self._agent_log_repository.get_agent_log(agent_id)
         except UnknownRecordError as err:
             logger.error(f"Error occurred while getting agent log: {err}")
```

Comment:
We should always prefer `logger.exception`, because it also includes traceback. The way it's done right now, traceback (the most important thing about the error) is missing